### ISSUE TITLE
Use last known sensor value

### ISSUE THREAD

**Issue by Kjelljk** [YorkshireIoT/ha-google-fit#88]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/YorkshireIoT/ha-google-fit/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

In config: "When there is no data available on the account for a sensor set to zero. If unchecked, set to unknown."

When sensor-data is set to unknown (or 0), and used in charts, the presentation gest missing areas/dottet. 

See attached picture.

Pink line is from HA-companion app (steps from user 1).

Blue line is from Fit-integration (steps from user 2). The line from Fit-integration becomes a dotted line because of the "unknown" data.

If no data is available, just keep last known value???

### Describe the solution you'd like

If no data, just keep last value. 

### Describe alternatives you've considered

Make templates to modify the sensor-data before use, but I think this could be an issue for more users.

### Additional context

![image](https://github.com/YorkshireIoT/ha-google-fit/assets/91701369/84d81d50-04a3-41b7-989b-601d571ca74e)

**Comment by Defozo** [comment]
I agree. There should be an option to keep the last value. This is also a problem when creating automation and wanting to know a sensor's previous known value.

**Comment by kc-sudo** [comment]
I agree with this. My sensors keep showing unknown. I would suggest to keep the previous value for a sensor till a new value is retrieved from the Google fit API

**Comment by fwmarcel** [comment]
I agree with this as well. My sensors show unknown from time to time. It would be wonderful if the values remain.

**Comment by YorkshireIoT** [comment]
There is an alpha release out where this should hopefully be done. Please give it a try and report back any issues as it's a fairly significant code change

**Comment by fwmarcel** [comment]
I have installed and tested the alpha release.
I noticed the following behavior.

During the "update phase", the last value remains unchanged. Wonderful. Thank you @YorkshireIoT  

The values are now only unknown after the start of Home Assistant until the interval has run once.
Wouldn't it make sense to query the values directly after the start?

The same behavior occurs when you change the interval. Then the values are also unknown for the time being.
